src/AsyncStore.ts

```diff
@@ -13,6 +13,7 @@ interface AsyncStore {
   isInitialized: () => boolean;
   getId: () => string | undefined;
   getShortId: () => string | undefined;
+  reset: (key?: string) => void;
```

Comment:
@samirsilwal Can we use reset to only reset the store? If we want to delete a key let's add a `del` method instead.

Reply:
Will make the changes